jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalUpcallHandler.java

```diff
@@ -71,19 +62,7 @@ public final class InternalUpcallHandler {
 	 * @return an internal upcall handler with the thunk address
```

Comment:
`@return` is not appropriate for a constructor; please remove that line and line 74.

Reply:
Removed as suggested above.